src/test/java/uk/gov/hmcts/ccd/domain/service/validate/DefaultValidateCaseFieldsOperationTest.java

```diff
@@ -107,6 +110,37 @@ void shouldValidate_when_organisation_has_correct_roles() throws Exception {
         );
     }
 
+    @Test
+    void shouldValidate_when_organisation_has_correct_roles_complex() throws Exception {
+
+        String orgPolicyReference1 = ORGANISATIONPOLICYFIELD_1 + "." + ORGANISATION_POLICY_ROLE;
```

Comment:
may be it would be better adding other cases complex to complex 3 levels aa.aaa.aaa  and see how the  componet .  componet . componet works

Reply:
this logic we have tested from UI, it is fine!!

Reply:
cut and paste , change values of the test . ok for me is the same . small comments :)